CHANGELOG.md

```diff
@@ -10,171 +10,218 @@ changes in the following format: PR #1234***
+- Fix for file deletion (PR #7284)
+#### User Accounts
+- Fix a false positive validation error when a new LORIS user is added with "Make 
+user name match email address" and "Generate new password". (PR #6803)
+- Fix to allow a superuser to create new users with customizable permissions. (#6770)
+- Indicate required fields (#6617)
+- Fix to show superuser the labels of each permission when editing a user (PR #7451)
 
 ### Tools
-- Fix fatal errors in delete_candidate.php tool. (PR #6805, #7275)
-- Fix fatal errors in fix_candidate_age.php (PR #7546)
-- New tool generate_candidate_externalids.php to fill external IDs for all candidates where a NULL value is found. (PR #7095)
-- New tool `populate_visits.php` to backpopulate visits from the `config.xml`, `session` table and `Visit_Windows` table into the `visit` and `visit_project_subproject_rel` (#7663)
+- Fix fatal errors in `delete_candidate.php` tool. (PR #6805, #7275)
+- Fix fatal errors in `fix_candidate_age.php` (PR #7546)
+- New tool `generate_candidate_externalids.php` to fill external IDs for all 
+candidates where a NULL value is found. (PR #7095)
+- New tool `populate_visits.php` to back-populate visits from the `config.xml`, 
+`session` table and `Visit_Windows` table into the `visit` and `visit_project_subproject_rel` (#7663)
 - Deprecation of the `populate_visit_windows.php` tool in favour of `populate_visits.php` (#7663)
 - Fixes a bug in the way that the data dictionary entries associated to MRI comments were named in the CouchDB database (PR #7082).
 
+
 ### Clean Up
-- Removal of unused variables and unnecessary branching from `getBattery()` and `getBatteryVerbose()` functions (PR #7167)
-- Removal of the violated_scans_edit permission (PR #6747)
-- Removal for the need of the `VisitLabel` section of the `config.xml` file. All Visit configurations and their association to projects are now in the database (#7663 & #7729)
-- Removal of unused variables and unnecessary branching from `getBattery()` and `getBatteryVerbose()` functions (PR #7167)
+- Removal of unused variables and unnecessary branching from `getBattery()` and 
+`getBatteryVerbose()` functions (PR #7167)
+- Removal of the `violated_scans_edit` permission (PR #6747)
+- Removal for the need of the `VisitLabel` section of the `config.xml` file. All 
+visit configurations and their association to projects are now in the database (#7663 & #7729)
 - Removal of references to Reliability module in Raisinbread (PR #6895)
 - Raisinbread visit stage inconsistency changed (PR #6896)
 - HRRT patch sourced to Raisinbread (PR #6897)
 - Improved consistency of RB data: field UploadLocation of table mri_upload now has an appropriate value (PR #7086)
+- The default value of the `ScannerID` field of the `mri_protocol` table is now 
+`NULL` instead of `0`. This means that if a protocol is valid on all the study's 
+scanners, then `ScannerID` of the protocol should be set to `NULL` (PR #7496)
+
```

Comment:
Should the new cleanup MySQL patches be mentioned here? Or maybe under Notes for existing projects...

Reply:
both is better than none... can you make sure to add it if it's missing (I dont really have the context)